sos/report/plugins/kdump.py

```diff
@@ -127,6 +145,22 @@ def setup(self):
             "/etc/default/kdump-tools"
         ])
 
+        try:
+            path = self.read_kdump_conffile()
+        except Exception:  # pylint: disable=broad-except
+            # set default path of coredir
+            path = "/var/crash"
+
+        self.add_dir_listing(path, recursive=True)
+        self.add_copy_spec(f"{path}/kexec_cmd")
+        self.add_copy_spec(f"{path}/kdump_lock")
+        self.add_copy_spec(f"{path}/*/dmesg*")
+        self.add_copy_spec(f"{path}/linux-image-*", sizelimit=2048, maxage=24)
```

Comment:
Worth calling just one `add_copy_spec` for the four files/globs?

Please add DCO to your commit message (add there line like "Signed-off-by: Arif Ali <arif-ali@ubuntu.com>").

Otherwise ack for the PR.

Reply:
@pmoravec It is better to make one add_copy_spec call for kexec_cmd, kdump_lock and dmesg* file.

Ah I don't how I forgot the signoff. I will update the pr.

Thanks for the review.